tests/nnstreamer_edge/runTest.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+##
+## SPDX-License-Identifier: LGPL-2.1-only
+##
+## @file runTest.sh
+## @author Yechan Choi <yechan9.choi@samsung.com>
+## @date 20 Jul 2022
+## @brief SSAT Test Cases for NNStreamer
+##
+if [[ "$SSATAPILOADED" != "1" ]]; then
+    SILENT=0
+    INDEPENDENT=1
+    search="ssat-api.sh"
+    source $search
+    printf "${Blue}Independent Mode${NC}"
+fi
+
+# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
+testInit $1
+
+# NNStreamer and plugins path for test
+PATH_TO_PLUGIN="../../build"
+PERFORMANCE=0
+
+check_edgesink=$(gst-inspect-1.0 --gst-plugin-path=${PATH_TO_PLUGIN} edgesink | grep EdgeSink)
+if [[ ! $check_edgesink ]]; then
+    echo 'Cannot find edge sink plugins. Skip tests.'
+    report
+    exit
+fi
+
+## @brief Execute file comparison test if both files exist
+function callCompareTestIfExist() {
+    if [[ ! -f "$1" ]]; then
+        echo "$1 don't exist. Skip $3($4)."
+    fi
+
+    if [[ ! -f "$2" ]]; then
+        echo "$2 don't exist. Skip $3($4)."
+    fi
```

Comment:
It's not skipping the tests.
```suggestion
    if [[ ! -f "$1" ]]; then
        echo "$1 don't exist. Skip $3($4)."
        return
    fi

    if [[ ! -f "$2" ]]; then
        echo "$2 don't exist. Skip $3($4)."
        return
    fi
```

Reply:
Updated.